src/aiortc/contrib/media.py

```diff
@@ -340,35 +325,29 @@ def player_worker(
                 )
  
                 if prediction_type != "keypoints":
-                    #TODO: using the interpolate function will result in stall of receiver in the pipeline
-                    '''
-                    driving = frame_to_tensor(img_as_float32(frame_array), device)
-                    driving_lr = F.interpolate(driving, lr_size).data.cpu().numpy()
-                    driving_lr = np.transpose(driving_lr, [0, 2, 3, 1])[0]
-                    driving_lr *= 255
-                    driving_lr = driving_lr.astype(np.uint8)
-
-                    lr_frame = av.VideoFrame.from_ndarray(driving_lr)
+                    frame_tensor = frame_to_tensor(img_as_float32(frame_array), device)
+                    lr_frame_array = resize_tensor_to_array(frame_tensor, lr_size, device)
+
+                    lr_frame = av.VideoFrame.from_ndarray(lr_frame_array)
                     lr_frame.pts = frame.pts
                     lr_frame.time_base = frame.time_base
```

Comment:
Don't we need an lr_frame.index also?

Reply:
I'm aware of the issue. We can't re-write the index of an av.VideoFrame, unfortunately.

Reply:
Can you make a note of it in a comment or something?

Reply:
done